src/main.rs

```diff
@@ -34,43 +34,59 @@ const DEFAULT_HELPER: &str = "aspect-credential-helper";
 #[command(version, about)]
 struct Args {
     /// SSH hostname to which to sync credential
-    #[arg(default_value_t = String::from("devbox"))]
+    #[arg(default_value = "devbox")]
     host: String,
 
     /// Aspect remote DNS name
-    #[arg(env = "ASPECT_REMOTE", default_value_t = DEFAULT_REMOTE.into(), long)]
+    #[arg(env = "ASPECT_REMOTE", default_value = DEFAULT_REMOTE, long)]
     remote: String,
 
     /// Aspect credential helper executable name
-    #[arg(env = "ASPECT_CREDENTIAL_HELPER", default_value_t = DEFAULT_HELPER.into(), long)]
+    #[arg(env = "ASPECT_CREDENTIAL_HELPER", default_value = DEFAULT_HELPER, long)]
     credential_helper: String,
 
     /// Force re-login even if the credentials are still valid
     #[arg(short, long)]
     force: bool,
 
-    /// Deprecated, do not use.
-    #[arg(short, long)]
-    _persist: bool,
-
     /// Use the session (rather than user) keyring on the VM
     #[arg(short, long)]
     session_keyring: bool,
 
-    /// Reuse existing socket (host has ControlMaster=auto and ControlPersist)
+    /// Create a temporary SSH control socket (if unset, this is automatically inferred)
+    #[arg(short, long, conflicts_with = "no_create_socket", default_missing_value = "true",
+          num_args = 0..=1, require_equals = true)]
+    create_socket: Option<bool>,
+
+    /// Do not create a temporary SSH control socket
+    #[arg(short = 'C', long, conflicts_with = "create_socket")]
+    no_create_socket: bool,
+
+    /// Reuse existing socket (Deprecated: use --create-socket=false instead)
```

Comment:
Is there any way you can just remove the deprecated arguments? As a new user, I think I would find them confusing, and now would be the time to do it. Any migration pain will have to come eventually, and might as well rip the band-aid off while the userbase is relatively small. Anyone who doesn't have the time to faff with it can always just not update?

Reply:
Yeah, currently my `users/mrdomino/reauth` uses that, and that reauth script is now recommended for folks but is not tightly version-controlled. I'll roll this change, then roll a change to the script, then roll a change that drops the old flags.